tests/acceptance/complete_review_offer_test.js

```diff
@@ -0,0 +1,51 @@
+import Ember from 'ember';
+import startApp from '../helpers/start-app';
+import FactoryGuy from 'ember-data-factory-guy';
+import TestHelper from 'ember-data-factory-guy/factory-guy-test-helper';
+import '../helpers/custom-helpers';
+import { module, test } from 'qunit';
+import '../factories/offer';
+import '../factories/item';
+import '../factories/packages_location';
+
+var App, offer1, reviewer, reviewerName, item1, item2, packages_location;
+
+module('In Review Offers', {
+  beforeEach: function() {
+    App = startApp({}, 2);
+    TestHelper.setup();
+    reviewer = FactoryGuy.make('user', { id: 3 });
+    offer1 = FactoryGuy.make("offer", { state:"under_review", reviewedBy: reviewer});
+    item1 = FactoryGuy.make("item", { state:"accepted", offer: offer1 });
+    item2 = FactoryGuy.make("item", { state:"accepted", offer: offer1 });
```

Comment:
We dont need to create 2 items and add it to offer in this case. We can create only 1 item and add it offer. :)

Reply:
Sure I will make the changes